tests/spec/context_spec.lua

```
@@ -0,0 +1,296 @@
+  get_node_text = function(node, bufnr)
+    local start_row, start_col, end_row, end_col = node.range()
+    local lines = vim.api.nvim_buf_get_lines(bufnr, start_row, end_row + 1, false)
+    return table.concat(lines, "\n")
+  end
+}
+
+-- Load the context module
+local context = require('caramba.context')
+
+describe("caramba.context", function()
+  
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)